pkgs/applications/office/marp/default.nix

```diff
@@ -1,37 +1,34 @@
-{ lib, stdenv, fetchurl, atomEnv, libXScrnSaver, gtk2 }:
+{ lib, stdenv, fetchurl  }:
 
 stdenv.mkDerivation rec {
   pname = "marp";
-  version = "0.0.14";
+  version = "1.4.2";
 
   src = fetchurl {
-    url = "https://github.com/yhatt/marp/releases/download/v${version}/${version}-Marp-linux-x64.tar.gz";
-    sha256 = "0nklzxwdx5llzfwz1hl2jpp2kwz78w4y63h5l00fh6fv6zisw6j4";
+    url = "https://github.com/marp-team/marp-cli/releases/download/v${version}/marp-cli-v${version}-linux.tar.gz";
+    sha256 = "sha256-URdpmuveb9xEyeHLABeyObZjT6XaXv9tPgrxH9XdBZk=";
   };
 
   unpackPhase = ''
-    mkdir {locales,resources}
-    tar --delay-directory-restore -xf $src
-    chmod u+x {locales,resources}
+    tar -xf $src
   '';
```

Comment:
Doesn't that work automatically?

Reply:
Do you mean, this command is the default `unpackPhase` implementation?
Removing this makes it error:
```
this derivation will be built:
  /nix/store/dsyrcfnvikari2428kvv633hdy43vg58-marp-1.4.2.drv
building '/nix/store/dsyrcfnvikari2428kvv633hdy43vg58-marp-1.4.2.drv'...
unpacking sources
unpacking source archive /nix/store/ki61hnvymiq78mzlzqz1jvs2kc95aam2-marp-cli-v1.4.2-linux.tar.gz
unpacker appears to have produced no directories
error: builder for '/nix/store/dsyrcfnvikari2428kvv633hdy43vg58-marp-1.4.2.drv' failed with exit code 1;
       last 3 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/ki61hnvymiq78mzlzqz1jvs2kc95aam2-marp-cli-v1.4.2-linux.tar.gz
       > unpacker appears to have produced no directories
       For full logs, run 'nix log /nix/store/dsyrcfnvikari2428kvv633hdy43vg58-marp-1.4.2.drv'.
```